apertium-kaa.kaa.lexc

```diff
@@ -30078,3 +30088,5522 @@ zuwla:zuwla V-TV ;
+Izmir% Super% Háweskerler% Ligasi:Izmir% Super% Háweskerler% Ligasi  NP-ORG ; !  
+Jáhán% ligasi:Jáhán% ligasi  NP-ORG ; !  
+Qubla% futbol% ligasi:Qubla% futbol% ligasi  NP-ORG ; !  
+Juma% Ligasi:Juma% Ligasi  NP-ORG ; !  
+Kataloniya% basketbol% ligasi:Kataloniya% basketbol% ligasi  NP-ORG ; !  
+Kipr% Birinshi% ligasi:Kipr% Birinshi% ligasi  NP-ORG ; !  
+Kipr% Ekinshi% ligasi:Kipr% Ekinshi% ligasi  NP-ORG ; !  
+Kipr% Tórtinshi% Ligasi:Kipr% Tórtinshi% Ligasi  NP-ORG ; !  
+Kipr% Úshinshi% Ligasi:Kipr% Úshinshi% Ligasi  NP-ORG ; !  
+Kolkata% futbol% ligasi:Kolkata% futbol% ligasi  NP-ORG ; !  
+Qoltıq% chempionlar% ligasi:Qoltıq% chempionlar% ligasi  NP-ORG ; !  
+Latviya% ligasi:Latviya% ligasi  NP-ORG ; !  
+Liga% kolledji:Liga% kolledji  NP-ORG ; !  
+Liga% Ligasi:Liga% Ligasi  NP-ORG ; !  
+Liganiń% Ekstraklasasi:Liganiń% Ekstraklasasi  NP-ORG ; !  
+Litva% Birinshi% Basketbol% Ligasi:Litva% Birinshi% Basketbol% Ligasi  NP-ORG ; !  
+Litva% Ekinshi% Basketbol% Ligasi:Litva% Ekinshi% Basketbol% Ligasi  NP-ORG ; !  
+Litva% ligasi:Litva% ligasi  NP-ORG ; !  
+Lombard% ligasi:Lombard% ligasi  NP-ORG ; !  
+Lotin% ligasi:Lotin% ligasi  NP-ORG ; !  
+Lyuksemburg% ligasi:Lyuksemburg% ligasi  NP-ORG ; !  
+Makedoniya% Birinshi% ligasi:Makedoniya% Birinshi% ligasi  NP-ORG ; !  
+Malayziya% Superligasi:Malayziya% Superligasi  NP-ORG ; !  
+Maribor% UEFA% Chempionlar% Ligasi:Maribor% UEFA% Chempionlar% Ligasi  NP-ORG ; !  
+Máwsim% UEFA% Chempionlar% ligasi:Máwsim% UEFA% Chempionlar% ligasi  NP-ORG ; !  
+Milliy% Basketbol% Ligasi:Milliy% Basketbol% Ligasi  NP-ORG ; !  
+Milliy% beysbol% ligasi:Milliy% beysbol% ligasi  NP-ORG ; !  
+Milliy% Beyzbol% Amerikalıq% Ligasi:Milliy% Beyzbol% Amerikalıq% Ligasi  NP-ORG ; !  
+Milliy% xokkey% ligasi:Milliy% xokkey% ligasi  NP-ORG ; !  
+Regionlıq% háweskerler% ligasi:Regionlıq% háweskerler% ligasi  NP-ORG ; !  
+Munitsipalitet% Beko% Basketbol% Ligasi:Munitsipalitet% Beko% Basketbol% Ligasi  NP-ORG ; !  
+Navarro% ACB% ligasi:Navarro% ACB% ligasi  NP-ORG ; !  
+NBA% Basketbol% Ligasi:NBA% Basketbol% Ligasi  NP-ORG ; !  
+NLB% ligasi:NLB% ligasi  NP-ORG ; !  
+Norvegiya% birinshi% ligasi:Norvegiya% birinshi% ligasi  NP-ORG ; !  
+Ochkolar% ligasi:Ochkolar% ligasi  NP-ORG ; !  
+Asiya% chempionler% ligasi:Asiya% chempionler% ligasi  NP-ORG ; !  
+Azerbaydjan% Premer%%-ligasi:Azerbaydjan% Premer%%-ligasi  NP-ORG ; !  
+ózbek% Ligasi:ózbek% Ligasi  NP-ORG ; !  
+PAF% ligasi:PAF% ligasi  NP-ORG ; !  
+Polsha% basketbol% ligasi:Polsha% basketbol% ligasi  NP-ORG ; !  
+Polsha% ligasi:Polsha% ligasi  NP-ORG ; !  
+Polsha% Úshinshi% Ligasi:Polsha% Úshinshi% Ligasi  NP-ORG ; !  
+Portugaliya% ligasi:Portugaliya% ligasi  NP-ORG ; !  
+Portugaliya% Premer%%-ligasi:Portugaliya% Premer%%-ligasi  NP-ORG ; !  
+Portugaliya% Superligasi:Portugaliya% Superligasi  NP-ORG ; !  
+Katar% ligasi:Katar% ligasi  NP-ORG ; !  
+Qızlar% ligasi:Qızlar% ligasi  NP-ORG ; !  
```

Comment:
Are the "ligasi" forms correct, or should it be "ligası"?  Also, you probably want a special category for `NP-ORG`s that end in possessive morphology.  E.g., `Qızlar ligası<n><loc>` is "Qızlar ligasında" not <s>"Qızlar ligasıda"</s>.  It should look similar to `N-COMPOUND-PX`.

Reply:
`NP-ORG-COMPOUND` from apertium-kaz is what you want, along with `N-COMPOUND-PX-COMMON`.  I believe the latter (or something like it) already exists in apertium-kaa.